packages/turbo-repository/js/index.d.ts

```diff
@@ -57,6 +57,7 @@ export class Workspace {
    */
   affectedPackages(
     files: Array<string>,
-    changedLockfile?: string | undefined | null
+    comparison?: string | undefined | null,
```

Comment:
Not blocking, but I feel like `from` or `base` is less confusing name.
Defer to Git wizard @NicholasLYang, if you're finding affected packages between `HEAD` and another commit, is there a standard name for what to call that other commit?

Reply:
@NicholasLYang I agree with @chris-olszewski - happy to refer to you on this and in my next PR on the `api` side, I'll be sure to rename it there too, to match.

Reply:
Probably base is best